mender-update/inventory.cpp

```diff
@@ -0,0 +1,171 @@
+// Copyright 2023 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+#include <mender-update/inventory.hpp>
+
+#include <sstream>
+#include <string>
+
+#include <common/api.hpp>
```

Comment:
Then this would be `common/http/api`?

Reply:
I had `http_api.hpp` for a while, but I guess it should use a subfolder, yes.